docker-compose.yaml

```diff
@@ -0,0 +1,67 @@
+version: '3.9'
+
+services:
+    php:
+        container_name: foundry_php
+        build:
+            context: .
+            dockerfile: ./docker/Dockerfile
+        depends_on:
+            mysql:
+                condition: service_healthy
+            postgres:
+                condition: service_healthy
+            mongo:
+                condition: service_healthy
+        volumes:
+            - .:/app
+        working_dir: /app
+
+    mysql:
+        container_name: foundry_mysql
+        image: mysql
+        command: --default-authentication-plugin=mysql_native_password
+        environment:
+            MYSQL_ROOT_PASSWORD: root
+            MYSQL_DATABASE: zenstruck_foundry
+        ports:
+            - ${MYSQL_PORT:-3306}:3306
+        healthcheck:
+            test: 'mysql -u root -proot -D zenstruck_foundry -e "SELECT 1"'
+            timeout: 120s
```

Comment:
this was the only way I found to make ci with macos pass. Otherwise I had "mysql is not healthy" errors